platforms/documentation/docs/src/snippets/providers/services/groovy/build.gradle

```diff
@@ -134,17 +134,6 @@ abstract class MyWorkerTask extends DefaultTask {
 tasks.register("myWorkTask", MyWorkerTask) {}
 // end::worker-executor[]
 
-// tag::file-system[]
```

Comment:
Do we want to provide any configuration-cache-friendly version of these snippets instead of just deleting them?

Reply:
>Do we want to provide any configuration-cache-friendly version of these snippets instead of just deleting them?

I've added some samples for ad-hoc tasks, but they aren't nice.